<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnQingInfoDialog</class>
 <widget class="QDialog" name="AnQingInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>案件信息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新增案件类别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="caseInPutEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>12</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>案件类别：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>401</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>处罚依据及处罚项</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>违反：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="caseWeiFanEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>依据：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="caseYiJuEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="caseChuFaEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>案件来源</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTextEdit" name="caseFromEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>案情</string>
   </property>
   <widget class="QTextEdit" name="caseAnQingBigEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>23</y>
      <width>379</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="caseAnQingSmallEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>139</y>
      <width>379</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>现场勘验</string>
   </property>
   <widget class="QTextEdit" name="caseCSIEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="text">
     <string>增  加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="text">
     <string> 删  除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modiButton">
    <property name="text">
     <string>修  改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="text">
     <string>退  出</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>600</y>
     <width>77</width>
     <height>50</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="okPushButton">
    <property name="text">
     <string>确 定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancerButton">
    <property name="text">
     <string>取 消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>caseWeiFanEdit</tabstop>
  <tabstop>caseYiJuEdit</tabstop>
  <tabstop>caseChuFaEdit</tabstop>
  <tabstop>caseFromEdit</tabstop>
  <tabstop>caseAnQingBigEdit</tabstop>
  <tabstop>caseAnQingSmallEdit</tabstop>
  <tabstop>caseCSIEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>modiButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancerButton</tabstop>
  <tabstop>caseInPutEdit</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
